tools/.lintstagedrc.json

```
@@ -1,8 +1,9 @@
 {
-  "../**/*.md": [
+  
+  "../**/*.{md,java,js,sh,yaml,yml}|*.{md,java,js,sh,yaml,yml}": [
```

Does this work to DRY it up?
```suggestion
  "(../**/|)*.{md,java,js,sh,yaml,yml}": [
```

Infact, why doesn't the simplified `../**/*.{md,java,js,sh,yaml,yml}` work? I thought that would match on the second part of the regex for a file within the same folder.

---

'../**/*' doesn't work because it uses relative paths, and '../**' only prefix-matches things strictly not on this directory

i will try the parentheses approach

---

Update: parentheses does not work sadly.